portugues/telefonica/ocupado.md

```diff
@@ -1 +1,5 @@
-Só dá ocupado.
\ No newline at end of file
+Só dá ocupado.
+
+[Você pega um ônibus até o centro para comprar a ultima versão XTree, mas você só tem Cr$ 5,00 na carteira.](centro/camelo.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.